docs/datatypes/writer.md

```diff
@@ -142,7 +142,7 @@ applies a bunch of Math operations, logging each one of them.
 import cats.data.Writer
 import scala.math.sqrt
 
-val writer1: Writer[String, Double] = Writer.value(5.0).tell("Initial value ")
+val writer1: Writer[String, Double] = Writer.value[String, Double](5.0).tell("Initial value ")
```

Comment:
This feels like a type inference regression from 2.12 -> 2.13.

Reply:
Yeah, seems so... I think it might be nice (in the future) to create a better version of `Writer.value` (and `WriterT.value`) with some sort of `PartialApply` involved which would allow to write something line this:
```
val writer1 = Writer.value[String](5.0).tell("Initial value ")
```

Reply:
Agree, we can do it bincompatibly probably but it would be source-breaking...